### DISCUSSION TITLE
F-Droid Repository

### DISCUSSION THREAD

**Discussion by 2br-2b** [BenTBCampbell/Software-Engineering#16] [General]
It would be nice if this app were in an F-Droid Repository. Even if it's not in the official one, this would allow for users to receive automatic updates for the app.

[F-Droid](f-droid.org/) is an Android app store made for Free and Open Source Software (FOSS), and anyone can make their own repository with their own apps. I have [my own repository](https://github.com/2br-2b/Fdroid-repo) that only has one app I made, but I could also add this to it.

Instructions for this can be found [here](https://f-droid.org/docs/Setup_an_F-Droid_App_Repo/).

**Comment by 2br-2b** [comment]
After receiving permission from @BenTBCampbell , I added the app to [my F-droid repository](https://github.com/2br-2b/Fdroid-repo).

To install this app and receive updates:

1. Install F-droid. F-droid is an app store for Free and Open Source (FOSS) applications. You can do this either by going to [https://f-droid.org/](https://f-droid.org/) on your phone or by scanning this qr code:

![https//fdroid.org/](https://gist.githubusercontent.com/2br-2b/7ba255bfb01032b050920d13942b5b59/raw/71dc87c161abbc307ec45f97706ce99eff3a8439/fdroid.svg)

2. After installing Fdroid, scan this qr code:

![https//rawgithubusercontentcom/2br2b/Fdroidrepo/master/fdroid/repo?fingerprint=90B5B0DDE20A84FB42CB960F41D279B0D3BB86578A42E773364A5534C0E8D27A](https://raw.githubusercontent.com/2br-2b/Fdroid-repo/master/QR%20code.svg)

There should be an option to open the link in F-droid. Do this and confirm that you want to add the repository.

If this doesn't work, choose to enter the repository data manually and paste https://raw.githubusercontent.com/2br-2b/Fdroid-repo/master/fdroid/repo?fingerprint=90B5B0DDE20A84FB42CB960F41D279B0D3BB86578A42E773364A5534C0E8D27A as the url.

3. Refresh F-droid's repositories by going to the "Latest" page and dragging from the top of the page
  
4. Search for "Oxword Fictionary" and choose to install it.

Every time you come out with a release, I will put the updated version onto my repository. After this, everyone who has gone through this process will receive a notification when an update is released and will be able to easily install it.

**Reply by BenTBCampbell** [reply]
Thanks @2br-2b, I copied these instructions to the wiki so people can find them there as well.
